File: Tugas2/java/proc.java
public class proc{
    private String name;
    private int pricep;

    // construct
    public proc() {}

    public proc(String name, int pricep){
        this.name = name;
        this.pricep = pricep;
    }
	
	//set and get
	public void setname(String name){
		this.name = name;
	}
	public String getname(){
		return this.name;
	}
	
	public void setpricep(int pricep){
		this.pricep = pricep;
	}
	public int getpricep(){
		return this.pricep;
	}

    public void printproc(){
        System.out.println("Nama : " + getname());
		System.out.println("Harga : " + getpricep());
    }

}

File: Tugas2/php/proc.php
<?php

class proc
{
	
	private $name = "";
	private $pricep = 0;
	
	
	//Construct
	public function __construct($name = "", $pricep = 0){
		$this->name = $name;
		$this->pricep = $pricep;
	}
	
	//Set dan get
	public function setname($name){
		$this->name = $name;
	}
	public function getname(){
		return $this->name;
	}
	
	public function setpricep($pricep){
		$this->pricep = $pricep;
	}
	public function getpricep(){
		return $this->pricep;
	}

    //Print Output
    public function printproc(){
		echo "Nama : ". $this->getname(). "<br/>";
		echo "Harga : ". $this->getpricep(). "<br/>";
	}
}
?>